library/src/main/java/com/chuckerteam/chucker/internal/support/FormatUtils.kt

```diff
@@ -94,6 +96,24 @@ internal object FormatUtils {
         }
     }
 
+    fun formatUrlEncodedForm(form: String): String {
+        return try {
+            if (form.isBlank()) {
+                return form
+            }
+            form.split("&").joinToString(separator = "\n") { entry ->
+                val keyValue = entry.split("=")
+                val key = keyValue[0]
+                val value = if (keyValue.size > 1) URLDecoder.decode(keyValue[1], "UTF-8") else ""
+                "$key: $value"
```

Comment:
```suggestion
entry.split("=").let { 
    (key, value) -> "$key: ${URLDecoder.decode(value, "UTF-8")}" 
}
```

You don't need extra variables as you can use a destructuring declaration.
Moreover you have to extend the catch below to include the `IndexOutOfBoundsException`

Reply:
My initial change had a destructuring declaration, but dropped it in favor of this to support what you can see in `testFormatUrlEncodedForm_noValues` (i.e., requests that need to send empty strings).

The current code should never throw an `IndexOutOfBoundsException`, since `split` always returns at least one string.

Reply:
Makes sense, let's drop my comment. Thanks for clarifying.